### ISSUE TITLE
Versions being removed from rubygems

### ISSUE THREAD

**Issue by rhomeister** [capistrano-plugins/capistrano-postgresql#45]
Hi guys (@NorseGaud),

It seems that old versions are being removed from rubygems, causing our build to break constantly. We updated to 4.6.0, which no longer exist in rubygems. Then we upgraded to 4.6.1 and the same happened. Inspecting rubygems (https://rubygems.org/gems/capistrano-postgresql/), it looks like all versions between 4.2 and 4.7 are now missing.

Does anyone know why this is happening?

**Comment by NorseGaud** [comment]
That was me. They were bad releases. Won’t happen again since I’m not pushing anything but 100% stable releases to rubygems anymore. Sorry about the inconvenience.

Also, releases under 4.6 never made it to rubygems.

**Comment by rhomeister** [comment]
OK, thanks for letting me know.

**Comment by rhomeister** [comment]
Hi guys,

It seems that this keeps happening. Our builds are failing because 4.8.0 doesn't exist anymore in Rubygems. Can you please fix this? It's causing some headaches....

Thanks,
Ruben

**Comment by NorseGaud** [comment]
https://github.com/capistrano-plugins/capistrano-postgresql/issues/49

While I intended to never cause this situation again, I messed up in replacing the 4.8.0 on rubygems (thought I could do it and was wrong) which had left over pry code within it. That deleted the gem. A version bump and gem push of 4.8.1 was successful, so I'm not sure how the wrong 4.8.0 got pushed.

With that said, 4.8.0 shouldn't have worked properly for anyone. It would throw someone into pry on each psql() helper method request.

Let me know if you see issues with 4.8.1. I'll double and triple check the gem before it's pushed from now on (even though I thought I did thorough testing before I pushed 4.8.0).
